src/runtime/Makefile

```diff
@@ -301,6 +307,10 @@ ifneq (,$(QEMUCMD))
     KERNELNAME = $(call MAKE_KERNEL_NAME,$(KERNELTYPE))
     KERNELPATH = $(KERNELDIR)/$(KERNELNAME)
 
+    KERNELSEVTYPE = compressed
+    KERNELSEVNAME = $(call MAKE_KERNEL_SEV_NAME,$(KERNELSEVTYPE))
```

Comment:
Where is `MAKE_KERNEL_SEV_NAME` defined?

Reply:
Fixed this.